it requires updates/deployments (#53)
intarchboard/draft-protocol-maintenance: draft-iab-protocol-maintenance.md

Sometimes what appear to be interoperability problems are symptomatic of issues
in protocol design. A community that is willing to make changes to the protocol,

> by revising or extending it and subsequently updating implementations
> and deploying them, makes the protocol better in the process.

Reviewer: This got a little convoluted.  Perhaps instead:

```suggestion
by revising or extending specifications then deploying those changes,
makes the protocol better.
```
